test/unit/content-src/components/TopSites.test.jsx

```diff
@@ -548,53 +548,53 @@ describe("<TopSite>", () => {
   });
 
   it("should render a TopSite", () => {
-    const wrapper = shallow(<TopSite link={link} />);
+    const wrapper = shallow(<TopSite link={link} intl={DEFAULT_PROPS.intl} />);
```

Comment:
you should be able to use `shallowWithIntl(<TopSite link={link} />)` here instead of adding the `intl` prop to each wrapper. similarly with all the ones below too